src/coreclr/vm/peimage.cpp

```diff
@@ -593,7 +594,7 @@ void PEImage::EnumMemoryRegions(CLRDataEnumMemoryFlags flags)
                         pCvInfo->path[MAX_LONGPATH - 1] = '\0';
 
                         //Find the filename from pdb full path
-                        char* fileName = strrchr(pCvInfo->path, '\\');
+                        char* fileName = strrchr(pCvInfo->path, DIRECTORY_SEPARATOR_CHAR_A);
```

Comment:
This is searching in a path embedded in .pdb file. I am not sure whether this change is right.

Reply:
If we embed on non-Windows, I would assume the paths could have `/`.

Reply:
It looks like that for managed assemblies we generate PDB files in "BSJB" format. Only native Windows libraries have the RSDS format. This code path is taken only for the latter (there is an "if" a bit above). So this change needs to be reserved and we would always get files with backslash separators here.